### PR TITLE
support encrypt when create logstore

### PR DESCRIPTION
**Pull request by HuiJing-C**

### CODE REVIEW

File: alicloud/resource_alicloud_log_store.go [hunk 1]
```diff
@@ -96,6 +96,47 @@ func resourceAlicloudLogStore() *schema.Resource {
 				Optional: true,
 				Default:  false,
 			},
+			"encrypt_conf": {
+				Type:     schema.TypeSet,
```
Comment (liuri1205) [inline alicloud/resource_alicloud_log_store.go:100]: 如果`encrypt_conf` 只是object类型, Schema 中加个 `MaxItems: 1`

**Review by liuri1205 — CHANGES_REQUESTED** [review]

File: alicloud/resource_alicloud_log_store_test.go [hunk 2]
```diff
@@ -37,12 +37,27 @@ func TestAccAlicloudLogStore_basic(t *testing.T) {
 					"name":        name,
 					"project":     "${alicloud_log_project.foo.name}",
 					"shard_count": "1",
+					// The following parameters are only used to enable logstore encryption when create logstore
+					// "encrypt_conf": []map[string]interface{}{
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_log_store_test.go:41]: If so, please add a testcase to support it.

File: website/docs/r/log_store.html.markdown [hunk 3]
```diff
@@ -43,12 +43,19 @@ The following arguments are supported:
...
-* `auto_split` - (Optional) Determines whether to automatically split a shard. Default to true.
+* `auto_split` - (Optional) Determines whether to automatically split a shard. Default to `true`.
 * `max_split_shard_count` - (Optional) The maximum number of shards for automatic split, which is in the range of 1 to 64. You must specify this parameter when autoSplit is true.
-* `append_meta` - (Optional) Determines whether to append log meta automatically. The meta includes log receive time and client IP address. Default to true.
-* `enable_web_tracking` - (Optional) Determines whether to enable Web Tracking. Default false.
+* `append_meta` - (Optional) Determines whether to append log meta automatically. The meta includes log receive time and client IP address. Default to `true`.
+* `enable_web_tracking` - (Optional) Determines whether to enable Web Tracking. Default `false`.
+* `encrypt_conf` (ForceNew,Optional) Encrypted storage of data, providing data static protection capability
```
Comment (xiaozhu36) [inline website/docs/r/log_store.html.markdown:52]: Please add a tag "Available in 1.124.0" for encrypt_conf, like `(ForceNew, Optional, Available in 1.124.0+)`

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]
Please rebase all of commits before submitting the the next commit

File: alicloud/resource_alicloud_log_store_test.go (hunk 2 above)
Comment (HuiJing-C) [reply alicloud/resource_alicloud_log_store_test.go:41]: 加密功能只在创建时支持，update不支持，所以测试只能放在这里

**Pull request closed without merging**